README.md

```diff
@@ -481,29 +481,11 @@ Install to the workbench's virtualenv by running the following command form the
 pip install -r requirements.txt
 ```
 
-In the main XBlock repository, create the following configuration file
-in `workbench/settings_mentoring.py` in the XBlock repository:
-
-```python
-from settings import *
-
-INSTALLED_APPS += ('mentoring',)
-DATABASES['default']['NAME'] = 'workbench.sqlite'
-```
-
-Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
-
-```bash
-$ ./manage.py syncdb --settings=workbench.settings_mentoring
```

Comment:
@bradenmacdonald Imho, as long as we have to go outside of the XBlock API, it can make sense to keep the current Django models approach - that likely reduces the amount of migration work and the potential unforseen pitfalls. It's still good to know that it's possible though, if we need to do this on 

Imho the real fix would be to add proper support for this - either within the XBlock API, via search, etc. Actually, it would be good to figure out what would be the right/canonical approach here, since this isn't the only change in the queue of upcoming work that will require accessing fields across different users (the other confirmed upcoming change will be to access answers in all mentoring MCQ fields from the current student and other students within a given cohort, to display the answers from everyone in the cohort in a single table for comparison). 

It would be worth posting about this on the ML imho, to see what Cale thinks is the right way to approach this - the last I've heard. (If you do, can you include the two use cases in the discussion?)